sorc/ncep_post.fd/MISCLN.f

```diff
@@ -1401,7 +1428,9 @@ SUBROUTINE MISCLN
             DO J=JSTA,JEND
               DO I=1,IM
                 GRID1(I,J)=Z1D(I,J)
-                IF (SUBMODELNAME == 'RTMA') FREEZELVL(I,J)=GRID1(I,J)
+                IF (SUBMODELNAME == 'RTMA' .OR. MODELNAME == 'FV3R') THEN
```

Comment:
@EdwardColon-NOAA May I ask the reason of adding model name "FV3R"?

Reply:
This additional conditional  model check (FV3R) was added in order to ensure that the FREEZELVL field would be accessible for the calculation of the SPC Hail parameter which needs to be an output product for the RRFS-based 3D-RTMA workflow.